src/main/java/seedu/duke/DukeGui.java

```diff
@@ -0,0 +1,133 @@
+package seedu.duke;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.stage.Stage;
+import org.w3c.dom.Text;
+import seedu.duke.cli.Command;
+import seedu.duke.cli.CommandException;
+import seedu.duke.cli.Parser;
+import seedu.duke.cli.commands.ByeCommand;
+import seedu.duke.tasks.TaskList;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+public class DukeGui extends Application {
```

Comment:
I think I kind of needing a javadoc here because this file is also the main process of the application